internal/cmd/trust/removerootkey/removerootkey.go

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package removerootkey
+
+import (
+	"os"
+	"strings"
+
+	"github.com/gittuf/gittuf/internal/cmd/common"
+	"github.com/gittuf/gittuf/internal/cmd/trust/persistent"
+	"github.com/gittuf/gittuf/internal/repository"
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	p            *persistent.Options
+	newRootKeyID string
```

Comment:
```suggestion
	rootKeyID string
```

this isn't a new key anymore.

Reply:
Fixed